data-source/jsonnet/england-wales/individual/blocks/identity-and-health/language.jsonnet

```diff
@@ -19,9 +19,29 @@ local walesOption = {
   value: 'English or Welsh',
```

Comment:
The 'values' for the England and Wales options could reference the new variables you've created here